### DISCUSSION TITLE
Module not found: Can't resolve 'react-hook-form'

### DISCUSSION THREAD

**Discussion by nataliakiselev** [react-hook-form/react-hook-form#3710] [General] [answered]
I've installed react-hook-form, yup and @hookform/resolvers, but getting  'Module not found: Can't resolve @hookform/resolvers' . Can't find the reason.

**Comment by nataliakiselev** [comment, marked as answer]
removing the current version and installing @hookform/resolvers@0.1.0 instead seems to sort this

**Reply by davidandrewfrey** [reply]
Thanks for posting – was having this issue today with 1.3.2

**Reply by bluebill1049** [reply]
you can use the latest beta `2.0.0` it solves this module issue.

**Reply by davidandrewfrey** [reply]
Thanks for the suggestion Bill – just gave 2.0.0-beta.11 a try and got this error:
```
./node_modules/@hookform/resolvers/dist/resolvers.module.js
Attempted import error: 'set' is not exported from 'react-hook-form' (imported as 't').
```

I'm using RHF 6.15.1 here.

I'll give this another shot sometime.

Also – just wanted to say THANK YOU!!! for RHF. It's really made working with complex forms easy (relatively speaking). We've been using it since v3.

**Reply by bluebill1049** [reply]
Tha's amazing to hear @davidandrewfrey ❤️ https://github.com/react-hook-form/resolvers/releases please use `v2.0.0-beta.6`

**Reply by davidandrewfrey** [reply]
Thanks again Bill – confirmed this is working now.

**Reply by nataliakiselev** [reply]
> you can use the latest beta `2.0.0` it solves this module issue.

Thanks for the reply, Bill. I've tried the latest hookform/resolvers 2.0.0 versions - got same error as described David above. Tried the v2.0.0-beta.6 - getting this error : Attempted import error: 'yupResolver' is not exported from '@hookform/resolvers'.  Updated yup and RHF, still getting same error. Should I use any particular versions combination of these to get it working?

**Reply by bluebill1049** [reply]
hey @nataliakiselev could you try 2.0.0-beta.3?

**Reply by nataliakiselev** [reply]
tried  the 2.0.0.-beta.3 - still getting the same error

**Reply by davidandrewfrey** [reply]
hey @nataliakiselev I have it working with RHF 6.15.1 and @hookform/resolvers 2.0.0-beta.6

**Reply by nataliakiselev** [reply]
It worked after changing the  { yupResolver } import  from @hookform/resolvers"   to "@hookform/resolvers/yup"

**Reply by bluebill1049** [reply]
cool 👍

**Reply by bluebill1049** [reply]
we are going to RC soon with all the beta version, and expect the final release coming soon as well.

**Reply by nataliakiselev** [reply]
Thanks, Bill!

**Reply by Lokesh-debug** [reply]
> It worked after changing the { yupResolver } import from @hookform/resolvers" to "@hookform/resolvers/yup"

can u help solving this I am getting same error
import { useForm } from "react-hook-form";
import {yupResolver} from "@hookform/resolvers/yup";
is this crct

**Reply by onataliaduran** [reply]
Hi, I using the RHF beta and the correct import but i'm still getting the error 

> Attempted import error: 'set' is not exported from 'react-hook-form' (imported as 'o').

**Reply by brunoeduardodev** [reply]
@onataliaduran try updating react hook form version, the @hookform/resolvers v2.0.0 only suport react-hook-form v7 >

**Reply by saadsaif97** [reply]
Thanks, I had an issue with 2.2.0 and resolved it with 0.1.0

**Comment by enlguy** [comment]
Not working...  Every post on this topic suggests a different version of @hookform/resolvers, yet NONE of them work for me.  I have downgraded and upgraded many times now, trying so many different versions.  Nothing is recognized.  I don't know if they forgot to change the docs after breaking something, but I have a simple import statement that is breaking my entire app, and haven't found a solution after an hour of scouring StackOverflow, github, and more.
